kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flights/FlightRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.good.patterns.flights;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class FlightRepository {
+    private Set<Flight> flights = new HashSet<>();
+
+    public FlightRepository() {
+        flights.add(new Flight("WAW", "", "GDN"));
```

Comment:
here is reason why I wrote previous comment, constructor looks strange when there is not intermediate landing